app/experimenter/nimbus-ui/src/components/PageRequestReview/DraftStatusOperations.tsx

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useCallback, useState } from "react";
+import Alert from "react-bootstrap/Alert";
+import { Config } from "../../hooks/useConfig";
+import { ReactComponent as Check } from "../../images/check.svg";
+import FormApproveConfirm from "./FormApproveConfirm";
+import FormApproveOrRejectLaunch from "./FormApproveOrRejectLaunch";
+import FormLaunchDraftToPreview from "./FormLaunchDraftToPreview";
+import FormLaunchDraftToReview from "./FormLaunchDraftToReview";
+import FormRejectReason from "./FormRejectReason";
+
+export enum DraftStatusOperationsStates {
```

Comment:
This replaced what looked like an increasing set of mutually-exclusive boolean toggles that I realized really wanted to be a simple state machine.

Reply:
Maybe I made this convention up in my head, and it's also an incredible nitpick, but should enums be singular?

Reply:
Hmm, I figured it should be a plural as the enum is a container for many members, but [Microsoft suggests singular](https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-1.1/4x252001(v=vs.71)?redirectedfrom=MSDN) (at least in this random page for .NET / C# that I found via Stack Overflow)

Reply:
I wouldn't worry about it, mostly just curious what your take on it was

Reply:
I always use singular for an enum 🤷‍♀️